src/main/resources/assets/extended_industrialization/lang/ru_ru.json

```
@@ -0,0 +1,322 @@
+{
+  "block.extended_industrialization.aluminum_tesla_winding": "Алюминиевая обмотка Теслы",
+  "block.extended_industrialization.annealed_copper_tesla_winding": "Отожжённая медная обмотка Теслы",
+  "block.extended_industrialization.blazing_essence": "Пламенная эссенция",
+  "block.extended_industrialization.bronze_bending_machine": "Бронзовый листогиб",
+  "block.extended_industrialization.bronze_composter": "Бронзовый компостер",
+  "block.extended_industrialization.bronze_solar_boiler": "Бронзовый солнечный котёл",
+  "block.extended_industrialization.bronze_waste_collector": "Бронзовый сборщик отходов",
+  "block.extended_industrialization.composted_manure": "Компостированный навоз",
+  "block.extended_industrialization.copper_tesla_winding": "Медная обмотка Теслы",
+  "block.extended_industrialization.distilled_water": "Дистиллированная вода",
+  "block.extended_industrialization.electric_alloy_smelter": "Электрическая печь сплавов",
+  "block.extended_industrialization.electric_bending_machine": "Электрический листогиб",
+  "block.extended_industrialization.electric_brewery": "Электрическая пивоварня",
+  "block.extended_industrialization.electric_canning_machine": "Электрический консервный аппарат",
+  "block.extended_industrialization.electric_composter": "Электрический компостер",
+  "block.extended_industrialization.electric_farmer": "Электрический фермер",
+  "block.extended_industrialization.electric_honey_extractor": "Электрический экстрактор мёда",
+  "block.extended_industrialization.electric_waste_collector": "Электрический сборщик отходов",
+  "block.extended_industrialization.electrum_tesla_winding": "Электрумовая обмотка Теслы",
+  "block.extended_industrialization.ev_tesla_receiver_hatch": "Приёмный люк Теслы экстремального напряжения",
```

Это **шлюз**, смотри внимательнее в [MI](https://github.com/AztechMC/Modern-Industrialization/blob/1.21.x/src/main/resources/assets/modern_industrialization/lang/ru_ru.json#L439).
```suggestion
  "block.extended_industrialization.ev_tesla_receiver_hatch": "Приёмный шлюз Теслы экстремального напряжения",
```